doc/usr/source/9_other/10_inductive_validity_core.rst

```diff
@@ -16,14 +16,17 @@ Options
 -------
 
 * ``--ivc_category {node_calls|contracts|equations|assertions}`` (default: all categories) -- Minimize only a specific category of elements, repeat option to minimize multiple categories
-* ``--ivc_enter_nodes <bool>`` (default ``false``\ ) -- Minimize elements of all the nodes (not only elements of the top-level node)
+* ``--ivc_only_main_node <bool>`` (default ``false``\ ) -- Compute an IVC over the elements of the main node only
+* ``--ivc_all <bool>`` (default ``false``\ ) -- Compute all the Inductive Validity Cores
+* ``--ivc_approximate <bool>`` (default ``false``\ ) -- Compute an approximation (superset) of an IVC
```

Comment:
I miss a comment saying  `Ignored if --ivc_all is true` for the ` --ivc_approximate` option